alure.pxd

```diff
@@ -442,7 +452,7 @@ cdef extern from '<AL/alure2.h>' namespace 'alure' nogil:
         float get_air_absorption_factor 'getAirAbsorptionFactor'() except +
 
         void set_gain_auto 'setGainAuto'(boolean, boolean, boolean) except +
-        # get_gain_auto
+        tuple[bool, bool, bool] get_gain_auto() except +
```

Comment:
We don't need this.  See the next 3 lines.

Reply:
Hey's it's not resolved.  This will **NOT** compile (there's no way to support `std::tuple` at the moment).